content/2_Bronze/Intro_Greedy.mdx

```diff
@@ -90,7 +90,8 @@ export const problems = {
 			'Very Hard',
 			true,
 			['Greedy'],
-			'Greedily increment/decrement Bessies speed to fit the conditions until her total distance exceeds K.'
+			'Greedily increment/decrement Bessies speed to fit the conditions until her total distance exceeds K.',
+			'usaco-989',
```

Comment:
Not linking properly.

Reply:
(link: https://usaco-guide-9hutey16s.vercel.app/bronze/intro-greedy?lang=cpp)

Reply:
Can you remove this solution altogether? I'll make another PR with my simulation code a bit later. :D

Reply:
Do you want to revert `content/2_Bronze/Intro_Greedy.mdx` and `solutions/USACO jan20-race.mdx`?

Reply:
Yeah.